app/src/pages/project/SpanFilterConditionField.tsx

```diff
@@ -271,8 +283,11 @@ export function SpanFilterConditionField(props: SpanFilterConditionFieldProps) {
   useEffect(() => {
     isConditionValid(deferredFilterCondition, projectId as string).then(
       (result) => {
-        if (!result?.isValid && result?.errorMessage) {
-          setErrorMessage(result.errorMessage);
+        if (!result?.isValid) {
+          if (result?.errorMessage) {
+            setErrorMessage(result.errorMessage);
+            return;
+          }
```

Comment:
hmm might be able to remove this can test, had something where filter was invalid but no error message which was causing issues, will test again to see